MeetingRoom cmdlets - clarification for Teams (#9129)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Disable-CsMeetingRoom.md

However, the Active Directory user account itself is not deleted.
This cmdlet was introduced in Lync Server 2013.

> **Note**: This cmdlet is not supported for managing Microsoft Teams Rooms, please use the methods mentioned in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation for the same.

Reviewer: ```suggestion
**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. We recommend you use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```

> **Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. We recommend you use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.

Reviewer 2: ```suggestion
**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. You must use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```

Reviewer 2: If they are not supported I think it's not a recommendation.
